PennyMe/ViewController.swift

```diff
@@ -294,9 +292,49 @@ class ViewController: UIViewController, UITextFieldDelegate {
             .compactMap { $0 as? MKGeoJSONFeature }
           let validWorks = features.compactMap(Artwork.init)
           artworks.append(contentsOf: validWorks)
+        // put IDs into a dictionary
+            for (ind, pin) in artworks.enumerated(){
+                pinIdDict[pin.id] = ind
+            }
         } catch {
           print("Unexpected error: \(error).")
         }
+        
+        // load from server
+        let link_to_json = "http://37.120.179.15:8000/server_locations.json"
```

Comment:
@jannisborn we now decided to leave the json file at that location on the server, right? Or will the link change? I'd only merge this PR once the link is fixed

Reply:
Yes, it will stay there!